docs/source/guide/export.md

```diff
@@ -388,3 +388,9 @@ First check the network access to your Label Studio instance when you send API r
 1. Check that you have mounted the same disk as your Label Studio instance. Then check your files' existence in Label Studio instance first. 
 
 2. Check `LOCAL_FILES_DOCUMENT_ROOT` environment variable in your Label Studio instance and add it to your accessing data script.
+
+3. Labels are sorted in alphabetical order, that is default behavior. If you want to modify that, please add `category` attribute to modify that behaviour. For example:
```

Comment:
How is it related to "### Question #2: I tried to access files and received a `FileNotFound` error." ? 
I think it should be different section with its own question.

Reply:
"How to modify order of labels for COCO and YOLO exports?"